documentation/src/main/asciidoc/topics/proc_using_mvn_archetypes.adoc

```diff
@@ -0,0 +1,58 @@
+[id='mvn_archetypes']
+= Creating {brandname} Projects
+{brandname} provides a set of Maven archetypes that you can use to generate
+skeleton projects with sample code and `pom.xml` files. You can then import
+these projects into your IDE and start building and running applications with
+{brandname} capabilities.
+
+.Prerequisites
+
+Before you can use the {brandname} archetypes, you need to do the following:
+
+. Open a terminal window.
+. Clone the {brandname} repository.
++
+----
+$ git clone git@github.com:infinispan/infinispan.git
+----
++
+. Change to the cloned repository and compile {brandname}.
++
+----
+$ cd infinispan
+$ mvn clean install -DskipTests
+----
+
+.Procedure
+
+. Navigate to the `archetypes` directory in the {brandname} repository.
++
+----
+$ cd archetypes
+----
++
+. Generate projects from {brandname} archetypes with the following command:
++
+----
+$ mvn archetype:generate -DarchetypeGroupId=org.infinispan.archetypes \
+    -DarchetypeArtifactId=<archetype-name> \ <1>
+    -DarchetypeVersion=<infinispan-version> <2>
+----
++
+<1> Creates projects from one of the following archetypes:
++
+* `client` Creates a sample Hot Rod Java client application.
+* `embedded` Creates a sample application that embeds {brandname}.
+* `store` Creates a sample implementation of a custom cache store.
++
+<2> Specifies the {brandname} version to use, for example:
++
+`-DarchetypeVersion={ispn_version}`
++
+. When prompted, define _artifactId_, _groupId_, and _version_ values for your project.
+. Import your new {brandname} project into your IDE.
+. Run your project application from the directory where the `pom.xml` file resides as follows:
```

Comment:
put a note saying the "store" archetype doesn't create any running code. (i.e. no Application.java)